### ISSUE TITLE
Accessibility example

### ISSUE THREAD

**Issue by jamesknelson** [frontarm/navi#11]
To support screenreaders, each branch should store a title under `data`. Then on navigation, the browser title should be updated, and an announcer div updated.

See https://twitter.com/noopkat/status/818240685715419136

This will need some changes:

- The documentation website will need to have the announcer div (it wouldn't hurt to use it for the documentation pages too)
- The react-junctions <Router> component should accept an onRouteChanged callback (or something similar), which we can use to update the browser title and announcer div

It would be nice to have all this built into react-junctions, but I can't see how to make it work:

- Multiple routes can be active at once, and deciding which one's title should be used is app-specific
- The announcer div needs to be available at page load, but junctions isn't loaded until later and probably doesn't know which div it is

**Comment by jamesknelson** [comment]
More thoughts:

The example is running inside a fake browser, but changing document title/the announcer div title will cause that change on the full page.

Either the fake browser needs to display its own title, or the documentation website needs to have titles set properly. Or both.
